# Case Study on ANOVA
XYZ Company has offices in four different zones. The company wishes to
investigate the following :
* The mean sales generated by each zone.
* Total sales generated by all the zones for each month.
* Check whether all the zones generate the same amount of sales.

Help the company to carry out their study with the help of data provided.

In [1]:
#importing dataset
import pandas as pd
import numpy as np

In [2]:
#reading file
d1=pd.read_csv('Sales_data_zone_wise.csv')
d1.head()

Month  Zone - A  Zone - B  Zone - C  Zone - D
0  Month - 1   1483525   1748451   1523308   2267260
1  Month - 2   1238428   1707421   2212113   1994341
2  Month - 3   1860771   2091194   1282374   1241600
3  Month - 4   1871571   1759617   2290580   2252681
4  Month - 5   1244922   1606010   1818334   1326062

In [3]:
d1.shape

(29, 5)

# 1.The mean sales generated by each zone

In [4]:
d1.describe()

Zone - A      Zone - B      Zone - C      Zone - D
count  2.900000e+01  2.900000e+01  2.900000e+01  2.900000e+01
mean   1.540493e+06  1.755560e+06  1.772871e+06  1.842927e+06
std    2.619401e+05  1.683899e+05  3.331937e+05  3.750165e+05
min    1.128185e+06  1.527574e+06  1.237722e+06  1.234311e+06
25%    1.305972e+06  1.606010e+06  1.523308e+06  1.520406e+06
50%    1.534390e+06  1.740365e+06  1.767047e+06  1.854412e+06
75%    1.820196e+06  1.875658e+06  2.098463e+06  2.180416e+06
max    2.004480e+06  2.091194e+06  2.290580e+06  2.364132e+06

# Conclusion:
### The above table shows the summary statistics for the given dataset. There are 29 entries. The mean value for each zone is:
* Zone A:1.54 Lakhs.
* Zone B:1.75 Lakhs.
* Zone C:1.77 Lakhs.
* Zone D:1.84 Lakhs.

The maximum value is:
* Zone A:2.00 Lakhs
* Zone B:2.09 Lakhs
* Zone C:2.29 Lakhs
* Zone D:2.36 Lakhs

# 2.Total sales generated by all the zones for each month.

In [5]:
d1['Total']=d1['Zone - A']+d1['Zone - B']+d1['Zone - C']+d1['Zone - D']
d1[['Month','Total']]

Month    Total
0    Month - 1  7022544
1    Month - 2  7152303
2    Month - 3  6475939
3    Month - 4  8174449
4    Month - 5  5995328
5    Month - 6  7151387
6    Month - 7  7287108
7    Month - 8  7816299
8    Month - 9  6703395
9   Month - 10  7128210
10  Month - 11  7032783
11  Month - 12  6111084
12  Month - 13  5925424
13  Month - 14  7155515
14  Month - 15  5934156
15  Month - 16  6506659
16  Month - 17  7149383
17  Month - 18  7083490
18  Month - 19  6971953
19  Month - 20  7124599
20  Month - 21  7389597
21  Month - 22  7560001
22  Month - 23  6687919
23  Month - 24  7784747
24  Month - 25  6095918
25  Month - 26  6512360
26  Month - 27  6267918
27  Month - 28  7470920
28  Month - 29  6772277

# 3.Check whether all the zones generate the same amount of sales

#### Using One-way ANOVA :
###### Level of significance, alpha = 0.05
###### Null Hypothesis, H0 : Same amount of sale for the four Zones.
###### Alternate Hypothesis, Ha : Different amount of sale for the four zones.

In [6]:
#impoting dataset
from scipy.stats import f_oneway

In [7]:
#oneway anova calculation
f_oneway(d1['Zone - A'],d1['Zone - B'],d1['Zone - C'],d1['Zone - D'])

F_onewayResult(statistic=5.672056106843581, pvalue=0.0011827601694503335)

##### Conclusion :
Here, p-value(0.0011827601694503335) is less than the alpha value (0.05). Hence we reject the null hypotheis and it can be concluded that the four zones have different sale.